apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -51,6 +51,9 @@ def prepare(element_html, data):
         element, required_attribs=required_attribs, optional_attribs=optional_attribs
     )
 
+    answer_name = pl.get_string_attrib(element, "answer-name")
```

Comment:
Other elements use answers-name, should it be updated to match other elements (with the old name undocumented and supported for compatibility)?

Reply:
Possibly? There's been some discussion of whether this element should even still exist, so it may not be worth maintaining at all.